packages/components/src/Slider/Slider.scss

```
@@ -26,6 +12,16 @@ $tc-slider-line-height: 24px;
 		background-color: transparent;
 		opacity: 0.4;
 	}
+	&__handler {
+		position: relative;
+	}
+	&__value {
+		position: absolute;
+		top: -25px;
+		color: $brand-primary;
+		font-size: $tc-slider-font-size;
```

```suggestion
		color: tokens.$coral-color-accent-text-strong;
		font: tokens.$coral-pahragraph-m-bold;
```
To check with the product designer (@maximenicoul) for the actual look and feel on the demo.